s/frontend/components/prepare-components.ts

```diff
@@ -10,6 +10,11 @@ import {themeComponents} from "../framework/theme-components.js"
 import {NamakaEmptyReview} from "./reviews/empty-review/index.js"
 import {NamakaReviewComment} from "./review/namaka-review-comment.js"
 import {NamakaHorizontalReview} from "./reviews/horizontal-five-star-review/index.js"
+<<<<<<< HEAD
```

Comment:
uh oh, we've got merge markers here

please fix this branch -- but be very careful not to accidentally lose any work, either yours, or other work